verifier.go

```diff
@@ -35,9 +38,15 @@ func NewVerifier(issuer string, keySource KeySource) *Verifier {
 	}
 }
 
-type verifyCfg struct{}
+type VerifyOpt func(*Verifier)
```

Comment:
@lstoll Do you think we need to keep `verifyCfg`? I couldn't really figure out a good reason to use it here. Our `ClientOpt` concept in `client.go` also just takes a `*Client`.

If you think there is a better way to accomplish this, though, let me know.

Reply:
This is a great point that I didn't consider. I'm going to change it back to using `verifyCfg` that is local to the `VerifyRaw` call.

Thank you for making me reconsider.